changelog.md

```diff
@@ -0,0 +1,18 @@
+# changelog
+
+All unreleased and notable changes to this project will be documented in this file.
+
+The format is like below:
+<pr title> ([#<pr id>](<pr link>), [@<your github name>](<your github homepage link>))
```

Comment:
This line is not rendered correctly, see: https://github.com/kubeedge/kubeedge/blob/c11266109adf97e050c208357ad64d6f16445d16/changelog.md